e2e/test/helpers/account.ts

```diff
@@ -16,13 +16,13 @@ export class Account implements Addressable {
         return new Wallet(this.privateKey, ETHERJS);
     }
 
-    async signWeiTransfer(counterParty: string, amount: BigNumberish, nonce?: number): Promise<string> {
+    async signWeiTransfer(counterParty: string, amount: BigNumberish, gasLimit: BigNumberish, nonce?: number): Promise<string> {
```

Comment:
Use a sensible default value for this new parameter to avoid the burden of all clients having to specify the gas when they don't care about it.